app/src/main/java/com/example/mytrainingpal/components/Break.kt

```diff
@@ -0,0 +1,197 @@
+                    Text(
+                        text = "Exercise ${currentExerciseIndex + 1} / ${exerciseList.size.toString()}"
+                    )
+                    Text(
+                        text = "Break",
+                        fontSize = 35.sp,
+                        fontWeight = FontWeight.Bold
+                    )
+                    Box(
+                        contentAlignment = Alignment.Center
+                    ) {
+                        BreakTimer(
+                            totalTime = breakTime * 1000L,
+                            //handleColor = Color.Green,
+                            inactiveBarColor = MaterialTheme.colors.primary,
+                            activeBarColor = MaterialTheme.colors.secondary,
+                            modifier = Modifier.size(180.dp)
+                        )
+                    }
+
+                    LinearProgressIndicator(
+                        progress = currentSet / totalSets.toFloat(),
+                        backgroundColor = MaterialTheme.colors.primary,
+                        color = MaterialTheme.colors.secondary,
+                        modifier = Modifier
+                            .fillMaxWidth()
+                    )
+                }
+
+            }
+            Row(
+                horizontalArrangement = Arrangement.SpaceAround,
+                modifier = Modifier
+                    .weight(1F, true)
+                    .fillMaxSize(),
+            ) {
+                if (currentExerciseSetCounter == exerciseList[currentExerciseIndex].second.sets && currentExerciseIndex == exerciseList.size) {
+                    Text(
+                        text = "You are almost done!",
+                        fontSize = 18.sp,
+                        fontWeight = FontWeight.Bold
+                    )
+                } else if (currentExerciseSetCounter == exerciseList[currentExerciseIndex].second.sets) {
+                    nextExerciseUp(
+                        gifPath = exerciseList[currentExerciseIndex + 1].first.pathToGif,
+                        exerciseName = exerciseList[currentExerciseIndex + 1].first.name,
+                        suggestedRepsForThatExercise = exerciseList[currentExerciseIndex].second.reps.split(
+                            ","
+                        )[0].toInt(),
+                        setsLeft = exerciseList[currentExerciseIndex].second.sets
+                    )
+                } else {
+                    nextExerciseUp(
+                        gifPath = exerciseList[currentExerciseIndex].first.pathToGif,
+                        exerciseName = exerciseList[currentExerciseIndex].first.name,
+                        suggestedRepsForThatExercise = exerciseList[currentExerciseIndex].second.reps.split(
+                            ","
+                        )[0].toInt(),
+                        setsLeft = exerciseList[currentExerciseIndex].second.sets - (currentExerciseSetCounter + 1)
```

Comment:
```suggestion
                        setsLeft = exerciseList[currentExerciseIndex].second.sets - currentExerciseSetCounter
```

Reply:
I would prefer this, since then the leftover sets shown in the break "next exercise" field reflect the actual leftover sets :)